securebanking-openbanking-uk-rs-simulator-server/src/main/java/com/forgerock/securebanking/openbanking/uk/rs/configuration/RsApplicationConfiguration.java

```diff
@@ -73,7 +77,12 @@ public Jackson2ObjectMapperBuilderCustomizer objectMapperBuilderCustomizer() {
             jacksonObjectMapperBuilder.featuresToDisable(DeserializationFeature.FAIL_ON_UNKNOWN_PROPERTIES);
             jacksonObjectMapperBuilder.featuresToEnable(MapperFeature.USE_BASE_TYPE_AS_DEFAULT_IMPL);
             jacksonObjectMapperBuilder.modules(new JodaModule());
-            jacksonObjectMapperBuilder.dateFormat(new SimpleDateFormat("yyyy-MM-dd'T'HH:mm:ssZZ"));
+            jacksonObjectMapperBuilder.featuresToEnable(DeserializationFeature.USE_BIG_DECIMAL_FOR_FLOATS);
+            jacksonObjectMapperBuilder.serializerByType(BigDecimal.class, new CustomObjectMapper.BigDecimalSerializer(BigDecimal.class));
+            jacksonObjectMapperBuilder.deserializerByType(BigDecimal.class, new CustomObjectMapper.BigDecimalDeserializer());
+            jacksonObjectMapperBuilder.deserializerByType(DateTime.class, new CustomObjectMapper.DateTimeDeserializer());
```

Comment:
For DateTime serialization, can we reuse: https://github.com/SecureBankingAccessToolkit/securebanking-openbanking-uk/blob/master/securebanking-openbanking-uk-obie-datamodel/src/main/java/uk/org/openbanking/jackson/DateTimeSerializer.java ?